test/drenv/addons/rook/rbd_mirror/test.py

```
@@ -41,85 +125,82 @@ def test_volume_replication(primary, secondary):
-        "--output=jsonpath={.spec.csi.volumeAttributes.imageName}",
-        context=primary,
-    )
 
+def _check_rbd_replication(primary, secondary, rbd_image, *rbd_args):
     print(f"rbd image {rbd_image} info in cluster '{primary}'")
-    out = _rbd("info", rbd_image, cluster=primary)
+    out = _rbd(*rbd_args, "info", rbd_image, cluster=primary)
     print(out)
 
     print(f"Waiting until rbd image {rbd_image} is created in cluster '{secondary}'")
     for i in range(60):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)